layouts/error.vue

```diff
@@ -1,14 +1,67 @@
 <template>
-  <div>
-    <h1>{{ error }}</h1>
+  <div class="container bg-block">
+    <h1 v-if="error.statusCode === 404" class="main-text">
```

Comment:
```suggestion
    <h1 v-if="error.statusCode === 404" class="is-size-1 has-text-weight-bold">
```
then delete the main text class in the styles

Reply:
if i do this i lose my font (which matches the rest of the website)